src/server/qgsserverparameters.cpp

```diff
@@ -531,7 +531,14 @@ QString QgsServerParameters::request() const
 
 QString QgsServerParameters::value( const QString &key ) const
 {
-  return value( QgsServerParameter::name( key ) ).toString();
+  if ( ! mParameters.contains( QgsServerParameter::name( key ) ) )
+  {
+    return mUnmanagedParameters[key];
```

Comment:
what happens if `key` is not in `mUnmanagedParameters` ?

Reply:
An empty string should be returned by qmap in this case. I added a unit test accordingly.